quinn-proto/src/crypto.rs

```diff
@@ -390,12 +398,26 @@ pub struct ZeroRttCrypto {
     cipher: &'static aead::Algorithm,
 }
 
-#[derive(Clone)]
 pub struct CryptoContext {
     local: CryptoState,
     remote: CryptoState,
     digest: &'static digest::Algorithm,
     cipher: &'static aead::Algorithm,
```

Comment:
*ring* keys store their `&'static aead::Algorithm` internally, so we can remove the `cipher` member.